bevrand.frontend/assets/js/randomizeCall.js

```diff
@@ -3,12 +3,62 @@ var randomizeList = '';
 var currentPlayList = 'tgif';
 var currentlySelectedPlayList = '';
 
+var token = "";
+var username = "";
+
 var config = {
     proxyHostname: 'https:' == document.location.protocol ? '' : 'http://localhost:4540'
 };
 
 $(document).ready(function () {
+    token = localStorage.getItem("jwt");
+    if (token) {
+        var jwtDecoded = parseJwt(token);
+        username = jwtDecoded['username'];
+        getUserPlaylist(function(userPlaylists) {
+            if (userPlaylists.length === 0) {
+                var playlistHtml = "<article>"
+                    + '<header>'
+                    + '<h3><a class="scrolly" href="/profile.html">You do not have any playlists! Create one in your Profile</a></h3>'
+                    + '</header >'
+                    + '</article >'
+
+                $('.reel').append(playlistHtml);
+            }
+            $('#navlinkLogin').hide();
```

Comment:
Perhaps extract these hides and shows to a seperate toggle function? The else contains the same type of logic only then with inverted hide and shows.

Reply:
Not part of this PR